freqtrade/exchange/binance.py

```diff
@@ -217,3 +224,58 @@ def load_leverage_tiers(self) -> Dict[str, List[Dict]]:
                     raise OperationalException(e) from e
         else:
             return {}
+
+
+
+    @retrier
+    @cached(cache=TTLCache(maxsize=100, ttl=10), lock=Lock())
+    def get_deslist_schedule(self):
+        try:
+
+            delist_schedule = self._api.sapi_get_spot_delist_schedule()
+
+            return delist_schedule
+        except ccxt.DDoSProtection as e:
+            raise DDosProtection(e) from e
+        except (ccxt.NetworkError, ccxt.ExchangeError) as e:
+            raise TemporaryError(
+                f'Could not get delist schedule {e.__class__.__name__}. Message: {e}') from e
+        except ccxt.BaseError as e:
+            raise OperationalException(e) from e
+
+    def get_spot_pair_delist_time(self, pair, refresh: bool = True) -> int | None:
+        """
+        Get the delisting time for a pair if it will be delisted
+        :param pair: Pair to get the delisting time for
+        :param refresh: true if you need fresh data
+        :return: int: delisting time None if not delisting
+        """
+
+        if not pair:
+            return
+
+        cache = self._spot_delist_schedule_cache
+        lock = self._spot_delist_schedule_cache_lock
+
+        schedule_pair = pair.replace('/', '')
```

Comment:
ccxt's `.markets` structure should have everything we need to convert symbol to pair (or viceversa).
i also think that the endpoint should be casting to pair already - so we get "DERP/BTC" out of the call - not DERPBTC. simply removing "/" is rather prone to errors in my experience.

it's often best to look at ccxt code on how they do it internally...